fineract-provider/src/main/java/org/apache/fineract/mix/data/ContextData.java

```diff
@@ -18,29 +18,18 @@
  */
 package org.apache.fineract.mix.data;
 
-public class ContextData {
-
-    private final String dimensionType;
-    private final String dimension;
-    private final Integer periodType;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
 
-    public ContextData(final String dimensionType, final String dimension, final Integer taxonomyType) {
-        this.dimensionType = dimensionType;
-        this.dimension = dimension;
-        this.periodType = taxonomyType.equals(MixTaxonomyData.BALANCESHEET) || taxonomyType.equals(MixTaxonomyData.PORTFOLIO) ? 0 : 1;
```

Comment:
This logic is missing where the object gets instantiated.

Reply:
corrected